test/SlowTests/Issues/RavenDB_10621.cs

```diff
@@ -9,15 +9,14 @@ namespace SlowTests.Issues
 {
     public class RavenDB_10621 : RavenTestBase
     {
-        [Fact]
+        [Fact(Skip = "Waiting for RavenDB-13830 to be resolved")]
```

Comment:
How does this generate this error?

Reply:
It doesn't it's an old test Michael wrote now it throws SO

Reply:
@ayende I think that this issue is related to an old issue that we had: https://github.com/dotnet/coreclr/issues/14672

Reply:
Yes, we wrote this test to make sure that we're correctly handling `InvalidProgramException` during indexing. Once https://github.com/dotnet/coreclr/issues/14672 got fixed we adjusted the test. See the following commit:

https://github.com/ravendb/ravendb/commit/de998b8b9d36b181dfc95f69b4733c3074a4bb23#diff-452a08ab6d945d7e51d8683da35507b3

Note that Pawel changed `[Fact64Bit]` to `[Fact]`. That's why we never saw any failures on 32bits before probably.

Investigation is reported in: https://issues.hibernatingrhinos.com/issue/RavenDB-13830